src/link.cpp

```diff
@@ -2387,11 +2387,17 @@ static void construct_linker_job_coff(LinkJob *lj) {
         lj->args.append(buf_ptr(buf_sprintf("-LIBPATH:%s", g->libc->crt_dir)));
 
         if (target_abi_is_gnu(g->zig_target->abi)) {
-            lj->args.append(buf_ptr(buf_sprintf("-LIBPATH:%s", g->libc->sys_include_dir)));
-            lj->args.append(buf_ptr(buf_sprintf("-LIBPATH:%s", g->libc->include_dir)));
```

Comment:
I'm actually not sure why we're passing any "include" directories as LIBPATH parameters, but just the same - is it intentional to omit the include directory here in the new code?

Reply:
doh, no was unintentional.